core/modules/url_filter.cc

```diff
@@ -0,0 +1,279 @@
+#include "url_filter.h"
+
+#include <rte_ip.h>
+
+#include <algorithm>
+
+#include "../utils/ether.h"
+#include "../utils/http_parser.h"
+#include "../utils/ip.h"
+
+using bess::utils::EthHeader;
+using bess::utils::Ipv4Header;
+using bess::utils::TcpHeader;
+
+const uint64_t TIME_OUT = 12e10;
```

Comment:
... or 120 * 1000 * 1000 * 1000. 120e9 is a floating point number so it may be slightly off after type casting.